WalletWasabi/Helpers/HttpMessageHelper.cs

```diff
@@ -483,7 +470,7 @@ public static void AssertValidHeaders(HttpHeaders messageHeaders, HttpContentHea
 				}
 			}
 			// Any Content-Length field value greater than or equal to zero is valid.
-			if (contentHeaders.Contains("Content-Length"))
+			if (contentHeaders != null && contentHeaders.Contains("Content-Length"))
 			{
 				if (contentHeaders.ContentLength < 0)
 					throw new HttpRequestException("Content-Length MUST be bigger than zero.");
```

Comment:
Could you change it to "larger than" instead of "bigger than"

Reply:
Sure.